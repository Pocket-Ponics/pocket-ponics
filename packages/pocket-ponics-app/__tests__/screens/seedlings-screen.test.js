import React from 'react'
import { shallow } from 'enzyme'

import SeedlingsScreen from '../../screens/seedlings-screen'

describe('Seedlings Screen', () => {
	test('renders correctly', () => {
		global.greenhouses = {}
		const tree = shallow(
			<SeedlingsScreen navigation={{ getParam: jest.fn() }}/>
		).html()
		expect(tree).toMatchSnapshot()
	})
})