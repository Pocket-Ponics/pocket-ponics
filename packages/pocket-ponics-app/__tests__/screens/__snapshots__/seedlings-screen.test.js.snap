// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Seedlings Screen renders correctly 1`] = `"<View style=\\"background-color:#472600;color:#FFFFFF;padding:20px;flex-direction:column;flex:1\\"><View style=\\"flex:1\\"><Image source=\\"1\\" style=\\"width:100%;height:175px;margin-top:30px;margin-bottom:30px;resize-mode:contain\\"></Image><Text style=\\"font-size:36px;font-weight:bold;margin-bottom:20px;color:#FFFFFF;align-items:center;text-align:center\\">Seedlings</Text><View style=\\"flex-direction:column;align-items:center\\"><View><Text style=\\"font-size:18px;padding-top:10px;color:#FFFFFF\\"><Text style=\\"font-weight:bold\\">Last Watered:</Text> NaN/NaN/NaN</Text><Text style=\\"font-size:18px;padding-top:10px;color:#FFFFFF\\"><Text style=\\"font-weight:bold\\">Next Water:</Text> NaN/NaN/NaN</Text><Text style=\\"font-size:18px;padding-top:10px;color:#FFFFFF\\"><Text style=\\"font-weight:bold\\">Transplant date:</Text> NaN/NaN/NaN</Text></View></View><View><View style=\\"background-color:#456E31;border-radius:5px;padding:10px;margin:10px;align-items:center;opacity:1\\"><Text style=\\"color:#FFFFFF;font-size:22px\\">Water Seedlings</Text></View></View></View></View>"`;
